web/modules/custom/usagov_benefit_finder/modules/usagov_benefit_finder_app/usagov_benefit_finder_page/templates/page--benefit-finder-life-event.html.twig

```diff
@@ -0,0 +1,142 @@
+{#
+/**
+ * @file
+ * Theme override to display a single page.
+ *
+ * The doctype, html, head and body tags are not in this template. Instead they
+ * can be found in the html.html.twig template in this directory.
+ *
+ * Available variables:
+ *
+ * General utility variables:
+ * - base_path: The base URL path of the Drupal installation. Will usually be
+ *   "/" unless you have installed Drupal in a sub-directory.
+ * - is_front: A flag indicating if the current page is the front page.
+ * - logged_in: A flag indicating if the user is registered and signed in.
+ * - is_admin: A flag indicating if the user has permission to access
+ *   administration pages.
+ *
+ * Site identity:
+ * - front_page: The URL of the front page. Use this instead of base_path when
+ *   linking to the front page. This includes the language domain or prefix.
+ *
+ * Page content (in order of occurrence in the default page.html.twig):
+ * - messages: Status and error messages. Should be displayed prominently.
+ * - node: Fully loaded node, if there is an automatically-loaded node
+ *   associated with the page and the node ID is the second argument in the
+ *   page's path (e.g. node/12345 and node/12345/revisions, but not
+ *   comment/reply/12345).
+ *
+ * Regions:
+ * - page.government_banner: USWDS Display Government site banner
+ * - page.header_top: Items for above the header region.
+ * - page.header: Items for the header region.
+ * - page.primary_menu: Items for the primary menu region.
+ * - page.secondary_menu: Items for the secondary menu region.
+ * - page.breadcrumb: Items for the breadcrumb region.
+ * - page.hero: USWDS Hero Banner
+ * - page.highlighted: Items for the highlighted content region.
+ * - page.help: Dynamic help text, mostly for admin pages.
+ * - page.content: The main content of the current page.
+ * - page.sidebar_first: Items for the first sidebar.
+ * - page.sidebar_second: Items for the second sidebar.
+ * - page.footer_menu: Items for the footer menu region.
+ * - page.footer: Items for the footer region.
+ * - page.footer_secondary: Items for below the footer secondary region.
+ *
+ * @see template_preprocess_page()
+ * @see html.html.twig
+ */
+#}
+<style>
+  @import url('https://fonts.googleapis.com/css2?family=Public+Sans:wght@200;400;700;800&family=Tilt+Warp&display=swap');
+</style>
+
+{% set lang = node.langcode.value %}
+{% set content_type = "" %}
+{% if node.type.0 != null %}
+  {% set content_type = node.type.0.target_id %}
+{% endif %}
+
+{% set term_name = "" %}
+{% if node.field_page_type.0 != null %}
+  {% set term_name = drupal_field('name', 'taxonomy_term', node.field_page_type.0.target_id)[0]['#context']['value'] %}
+{% endif %}
+
+{% set show_sidebar = true %}
+{% if term_name == "Navigation Cards Page" or term_name == "Life Events" or term_name == "Life Events Landing Page" %}
```

Comment:
I'm seeing a bunch of lines here we can probably simplify, since we already know what type of page this is.

Reply:
This template is only used for life event page.
We will simplify this template, removing these lines.